library/mongo-express

```diff
@@ -1,26 +1,28 @@
-# this file is generated via https://github.com/mongo-express/mongo-express-docker/blob/0b9b935b188caa945834a27f7475a059c4e60b13/generate-stackbrew-library.sh
+# this file is generated via https://github.com/mongo-express/mongo-express-docker/blob/dc51a0d22c492699b53830e30d311bb80c8310f1/generate-stackbrew-library.sh
 
 Maintainers: Nick Cox <nickcox1008@gmail.com> (@knickers),
              John Steel <john@jskw.dev> (@BlackthornYugen)
+
```

Comment:
I think it is this empty line that is tripping it up. With it, it considers just the `Maintainers:` above as the "global" section and then the `GitRepo` + `GitCommit` just below as a new item with no `Tags` (which it then complains about missing tags).

Reply:
Oh right -- let me try without that.

Reply:
Now it doesn't like my tags now. Says there isn't a ":" on that line? 

![image](https://github.com/docker-library/official-images/assets/3138467/db101be2-b2b5-4374-b899-07cb895a1390)

Reply:
Oh, that's exactly what's wrong. I'll double check the generate script. Fix it manually for now.